app/components/bali/stepper/index.scss

```diff
@@ -0,0 +1,83 @@
+$step-marker-border-size: 0.5rem;
+$step-marker-size: 2rem;
+$step-details-padding-top: 0.2rem;
+$step-connector-height: 0.2rem;
+
+.stepper-component {
+  display: flex;
+  flex-wrap: wrap;
+
+  .step-component {
+    margin-top: 0;
+    position: relative;
+    flex-grow: 1;
+    flex-basis: 0;
+
+    &:before {
+      background: linear-gradient(to left, $grey-lighter 50%, $primary 50%);
```

Comment:
I don't see the variable `$primary` declared anywhere. Please declare it in the variables.scss file.

Reply:
it's part of the bulma variables, which are already included